src/main/java/core/basesyntax/dao/StorageDaoImpl.java

```diff
@@ -22,4 +22,13 @@
         }
         return null;
     }
+
+    @Override
+    public void removeByLogin(String login) {
+        User user = get(login);
+        if (user != null) {
```

Comment:
Since the `get` method should return an `Optional<User>`, you will need to update this logic to handle the `Optional`. Consider using `ifPresent` or `isPresent` methods to check if a user is present before attempting to remove it.